ietf/meeting/models.py

```diff
@@ -385,17 +385,19 @@ def uses_notes(self):
         if self.type_id != 'ietf':
             return True
         num = self.get_number()
-        return num != None and num >= 108
+        return num is not None and num >= 108
 
     def has_recordings(self):
         if self.type_id != 'ietf':
             return True
         num = self.get_number()
-        return num != None and num >= 80
+        return num is not None and num >= 80
 
     def has_chat_logs(self):
+        if self.type_id != 'ietf':
+            return True;
```

Comment:
I think this should be `False` to preserve the behavior? For a non-ietf meeting, `self.get_number()` returns `None`, so it used to return `False`. (If it should be `True` or is a don't care, just say so, I'm just pointing out the change.)

Reply:
I think what's here now matches current and desired behavior. What existed earlier in this branch may have been off base about whether interims had chat logs, but the do (and always have).